tests/components/zwave_js/test_api.py

```diff
@@ -1082,3 +1132,86 @@ async def test_data_collection(hass, client, integration, hass_ws_client):
 
     assert not msg["success"]
     assert msg["error"]["code"] == ERR_NOT_LOADED
+
+
+async def test_abort_firmware_update(
+    hass, client, multisensor_6, integration, hass_ws_client
+):
+    """Test that the abort_firmware_update WS API call works."""
+    entry = integration
+    ws_client = await hass_ws_client(hass)
+
+    client.async_send_command_no_wait.return_value = {}
+    await ws_client.send_json(
+        {
+            ID: 1,
+            TYPE: "zwave_js/abort_firmware_update",
+            ENTRY_ID: entry.entry_id,
+            NODE_ID: multisensor_6.node_id,
+        }
+    )
+    msg = await ws_client.receive_json()
+    assert msg["success"]
+
+    assert len(client.async_send_command_no_wait.call_args_list) == 1
+    args = client.async_send_command_no_wait.call_args[0][0]
+    assert args["command"] == "node.abort_firmware_update"
+    assert args["nodeId"] == multisensor_6.node_id
```

Comment:
Please test that we handle wrong node id and unloaded entry for all new commands. See existing tests.

Reply:
tested all three failure scenarios: https://github.com/home-assistant/core/pull/50390/files#diff-de558ecb628ae08d31043b11187b4333792ff053a0ae92f9c406f31a34d1be9aR761-R794

Reply:
That only tests the firmware upload view. It doesn't test the new websocket commands for wrong node is and unloaded entry.

Reply:
oh sorry you are talking about the websocket commands. Since I am using the decorator to retrieve entry and node, do I need to retest? I have a test for the decorator already

Reply:
Yes, we need to test each command. If we would remove the decorator for this command we want the tests to fail.